### ISSUE TITLE
[XamarinProfiler-MAC]: Accessibility: Keyboard: MAS2.1.1: Cannot able to navigate backward using Shift+TAB from Zoom(-) scroll button of profiling table.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#445]
### Pre-Requisite:
"1. Install Xamarin Profiler.
2. Launch Visual Studio.
3. Create a project and build the solution.
4. Navigate to Run Menu Tab and Invoke it. Navigate to ""Start Profiling"" option and invoke it.
5. This will invokes ""Xamarin Profiler"" application."
### Steps to Reproduce
Scenario 1:
1. Follow the pre-requisite to launch the xamarin profiler application.
2. Select the Available instruments to "Time Profiler" and invoke next button. This will invokes Common options window.
3. Navigate to "Start Profiling" button and invoke it. This will launch Time Profiler profiling page of Call Tree. 
4. Navigate to "Zoom (-)" scroll button and press "Shift+Tab"

Scenario 2:
1. Follow the pre-requisite to launch the xamarin profiler application.
2. Select the Available instruments to "Allocations" and invoke next button. This will invokes Common options window.
3. Navigate to "Start Profiling" button and invoke it. This will launch Allocations profiling page.
4. Navigate to "Zoom (-)" scroll button and press "Shift+Tab"

### Expected Behavior
Should be able to traverse backward using Shift+TAB
### Actual Behavior
Cannot able to navigate backward using Shift+TAB from Zoom(-) scrool button of profiling table. Keyboard focus remains at Zoom(-) button even after pressing "Shift+Tab"

### User Impact
User will not be able to traverse backward.
### Screenshot
![zoom -](https://user-images.githubusercontent.com/22956130/41720390-89f9abfa-7580-11e8-9e6e-3fb711d359bc.png)
![zoom - allocations](https://user-images.githubusercontent.com/22956130/41722390-0aab09ba-7586-11e8-9193-b1db53b022ad.png)

### Environment
macOS High Sierra OSX version: 10.13.4
Visual Studio for Mac Enterprise version: 7.5.2(build 40)
Xamarin Profiler version: 1.6.3-400

**Comment by WiproPSAccTest** [comment]
Please add the following tags to this bug:
A11yMAS
Accessibility
MAS2.1.1
Xamarin-Profiler-MAC
Wipro
